### PR TITLE
Closes #641: Analysis chart as table in accordance with WCAG

### LINKED ISSUE #641: Analysis chart as table in accordance with WCAG
http://accessibility.psu.edu/images/charts/

### PR DESCRIPTION
**Pull request by lukdumi**

### CODE REVIEW

File: saos-webapp/src/main/webapp/WEB-INF/static/stylesheet/scss/_wcag.scss [hunk 1]
```diff
@@ -37,4 +37,17 @@
...
+    position: absolute; 
+    clip: rect(1px 1px 1px 1px); /* for IE6, IE7 */
+    clip: rect(1px, 1px, 1px, 1px);
+    margin: -1px;
+    border: 0; 
+    padding: 0; 
+    width: 1px;
+    height: 1px ; 
+    overflow: hidden;
+    diplay: block;
```
Comment (madryk) [inline saos-webapp/src/main/webapp/WEB-INF/static/stylesheet/scss/_wcag.scss]: did you mean `display`?

File: saos-webapp/src/main/resources/message/common.properties [hunk 2]
```diff
@@ -336,7 +336,8 @@ analysis.button.generateChart = Generate chart
 analysis.button.addPhrase = Add phrase
 analysis.button.deletePhrase = Delete phrase
 analysis.button.deletePhrase.iconAlt = Ikona usuwająca pole szukane hasło
-analysis.button.exportToCsv = Exportuj do pliku csv
```
Comment (madryk) [inline saos-webapp/src/main/resources/message/common.properties]: I guess that you removed this label by mistake. It is still used.

File: saos-webapp/src/main/webapp/WEB-INF/static/stylesheet/scss/_wcag.scss (hunk 1 above)
Comment (lukdumi) [inline saos-webapp/src/main/webapp/WEB-INF/static/stylesheet/scss/_wcag.scss]: :+1:

File: saos-webapp/src/main/resources/message/common.properties (hunk 2 above)
Comment (lukdumi) [inline saos-webapp/src/main/resources/message/common.properties]: :-1:

File: saos-webapp/src/main/webapp/WEB-INF/static/javascript/analysis.js [hunk 3]
```diff
@@ -930,43 +964,82 @@ var initAnalysisJs = function() {
     /**********************************************************************************************/    
 
     
-    var xtickPeriodFormatters = {};
+    /**
+     * Formats the given xtick contains a period of type pl.edu.icm.saos.common.chart.value.TimePeriod
```
Comment (madryk) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/analysis.js]: `Formats the given xtick that ...` or even without that part after `contains` (`Formats the given xtick`)?

Comment (lukdumi) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/analysis.js]: ok, i've added 'that'.

**Pull request closed without merging**